WootricSDK/WootricSDK/WTRApiClient.m

```diff
@@ -274,7 +274,7 @@ - (void)checkEligibility:(void (^)())eligible {
 
   NSURL *url = [NSURL URLWithString:baseURLString];
   NSMutableURLRequest *urlRequest = [NSMutableURLRequest requestWithURL:url];
-  [urlRequest setValue:@"Wootric-Mobile-SDK" forHTTPHeaderField:@"USER_AGENT"];
+  [urlRequest setValue:@"Wootric-Mobile-SDK" forHTTPHeaderField:@"User-Agent"];
```

Comment:
@marinhero I did kind of find a way to do it, but it will probably take some time and Jess's needs this asap.

XCTestExpectation: http://nshipster.com/xctestcase/
With this you can create iOS tests with actual asynchronous calls. That's why I'm saying to would take more time.